addons/account/models/account_account.py

```diff
@@ -138,6 +138,9 @@ def _check_account_type_unique_current_year_earning(self):
                                help="If set, this account will belong to Non Trade Receivable/Payable in reports and filters.\n"
                                     "If not, this account will belong to Trade Receivable/Payable in reports and filters.")
 
+    # Form view: show code mapping tab or not
+    display_mapping_tab = fields.Boolean(default=lambda self: len(self.env.user.company_ids) > 1, store=False)
```

Comment:
It's really a shame that we need this when both `allowed_company_ids` and `current_company_id` are available from the client :frowning_face:

Reply:
It is actually ok, I'm not sure why I understood this as `self.env.companies` :facepalm: 

My comment regarding `allowed_company_ids` didn't make sense at all